lib/service.test.js

```diff
@@ -41,14 +41,15 @@ describe('lib/service', () => {
 
     /**
      * @param {string} request_token
+     * @param {string|null} command
      * @param {string} color_level
      * @param {string} cwd
      * @param {string[]} argv
      * @param {string} [text]
      */
-    function send(request_token, color_level, cwd, argv, text) {
+    function send(request_token, command, color_level, cwd, argv, text) {
       const chunks = [
-        `["${request_token}",${color_level},`,
+        `["${request_token}",${command ? `"${command}"` : null},${color_level},`,
```

Comment:
Thank you for separating the color level from the command. We could actually simplify this further by always sending a command, e.g. `PING`, `SHUTDOWN` and `LINT`.

Reply:
Extracted all commands into constants, also added command for lint

Reply:
The `command` is non-optional now. The conditional can be removed.